### ISSUE TITLE
New Pull request "HTTP Referrer" and "User-Agent"

### ISSUE THREAD

**Issue by Flauschi1981** [gorhill/uBlock#416]
Hi 

can you plz add a new privacy setting fuktion that control Firefox how to send Referer http header?? 

many thx ( sry for my bad english)

**Comment by pid1** [comment]
This can be done with uMatrix. If you don't want uMatrix to perform any other functions, you can add a default allow all policy.

**Comment by akks1981** [comment]
(thx pid)

Hi,

can you plz import the privacy fuktion "HTTP Referrer" and  "User-Agent" from uMatrix to uBlock? 


Big Thx :-)

**Comment by anewuser** [comment]
> Submit bugs/issues only. 
> Do NOT:    Submit design ideas.
> Do NOT:    Submit feature requests.
> Do NOT:    Submit "revolutionary ideas".

https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md

**Comment by akks1981** [comment]
New Pull request dont work! but thx for the info...

**Comment by crssi** [comment]
I vote for a "HTTP Referrer" :)

**Comment by migueldemoura** [comment]
The feature you want is already present in [uMatrix](https://github.com/gorhill/uMatrix). Install it and add the following rules:

```
referrer-spoof: * true
* * * allow
```

You can set the user agent from the settings page.

@gorhill, please close this

**Comment by anewuser** [comment]
@migueldemoura 

This repository has been **abandoned**, and the main developer (gorhill) has no control over it anymore.

Please use these links for the actively developed extension you should be using - **uBlock Origin**:

## Download
* Official: [Firefox](https://addons.mozilla.org/addon/ublock-origin/), [Chrome](https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm), [Opera](https://addons.opera.com/extensions/details/ublock/)
* Acknowledged by the main developer: [Microsoft Edge](https://github.com/nikrolls/uBlock-Edge/#ublock-origin-beta-for-microsoft-edge), [Safari](https://github.com/el1t/uBlock-Safari#ublock-originfor-safari)

## Help
* Make sure you're using the latest version of uBlock Origin and **[read the manual](https://github.com/gorhill/uBlock/wiki)** before asking for help
* If you still need help or want to discuss features, **[use the official Discourse forum](https://discourse.mozilla-community.org/t/support-ublock-origin/6746)**, the [Wilders Security Forums](https://www.wilderssecurity.com/threads/ublock-a-lean-and-fast-blocker.365273/page-99999999999), or the [subreddit](https://www.reddit.com/r/uBlockOrigin/). Don't open issues on Github unless you've found a serious problem that makes the extension stop working altogether. Also, use the specific repositories for  [Edge](https://github.com/nikrolls/uBlock-Edge/)  and [Safari](https://github.com/el1t/uBlock-Safari) to report issues with them.
